Backend/routes/auth.js

```diff
@@ -59,11 +62,63 @@ router.get(
 router.get(
   "/google/verify",
   passport.authenticate("google", { failureRedirect: "/" }),
-  (_req, res) => {
-    res.redirect("/");
+  (req, res) => {
+    if (!req.user.ID_No) {
+      res.redirect("/auth/google/addId");
+    } else {
+      res.redirect("/");
+    }
   },
 );
 
+router.get("/google/addId", (req, res) => {
+  const token = jwt.sign({ id: req.user._id }, secretKey);
+
+  res.redirect(`http://localhost:3000/register/google/${token}`);
```

Comment:
It will create problems while deploying the application. Hence we should store it in .env so that we can adjust it as per our requirements.
```suggestion
  res.redirect(`${process.env.FRONTEND_URL}/register/google/${token}`);
```

Reply:
We will change that later, currently to keep testing easier, I suggest keeping it hardcoded